command-expansion-server/src/app.ts

```diff
@@ -8,11 +8,12 @@ import * as ampcs from "@nasa-jpl/aerie-ampcs";
 import { processDictionary } from "./packages/lib/CommandTypeCodegen.js";
 import { getActivityTypescript } from "./getActivityTypescript.js";
 import { getCommandTypescriptTypes } from "./getCommandTypescriptTypes.js";
+import { ErrorWithStatusCode } from "./utils/ErrorWithStatusCode.js";
 
 const PORT: number = parseInt(getEnv().PORT, 10) ?? 3000;
 
 const app: Application = express();
-app.use(bodyParser.json({ limit: "25mb" }));
+app.use(bodyParser.json({ limit: '25mb' }));
```

Comment:
Some spurious quote changes from double to single quotes seem to be cluttering the diff for this PR. Perhaps you guys have different settings on your autoformatters?

Reply:
In this case, this change was carried over from a commit by @dyst5422/@jdylanstewart. I think we're all onboard with using single quotes when possible for string literals that aren't expected to have interpolation, so I'm not expecting much code churn between single <-> double quotes.

Reply:
I am using Prettier in vscode which @camargo recommended to me for Aerie-UI. 

I followed this document to set it up. 
https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code

Might be worth keeping checking out

Reply:
I was planning on adding in a .prettierrc file to configure prettier for single quotes etc, just haven't wanted to clutter up our useful PRs with it.